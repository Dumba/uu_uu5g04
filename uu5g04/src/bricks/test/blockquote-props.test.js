/**
 * Copyright (C) 2019 Unicorn a.s.
 * 
 * This program is free software; you can use it under the terms of the UAF Open License v01 or
 * any later version. The text of the license is available in the file LICENSE or at www.unicorn.com.
 * 
 * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
 * the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See LICENSE for more details.
 * 
 * You may contact Unicorn a.s. at address: V Kapslovne 2767/2, Praha 3, Czech Republic or
 * at the email: info@unicorn.com.
 */

import React from 'react';
import UU5 from "uu5g04";
import "uu5g04-bricks";
import TestTools from "../../core/test/test-tools.js";
import renderer from 'react-test-renderer';

const TagName = "UU5.Bricks.Blockquote";

const CONFIG = {
  mixins: [
    "UU5.Common.BaseMixin",
    "UU5.Common.ElementaryMixin",
    "UU5.Common.ColorSchemaMixin",
    "UU5.Common.ContentMixin",
    "UU5.Common.NestingLevelMixin",
    "UU5.Common.PureRenderMixin"
  ],
  props: {
    background: {
      values: [true, false]
    },
    alignment: {
      values: ["left", "right"]
    },
    footer: "My Footer Content"
    ,
    footerAlignment: {
      values: ["left", "right"]
    },
    noSpacing: {
      values: [true, false]
    }
  },
  requiredProps: {},
  opt: {
    enzymeToJson: false
  }
};


describe(`${TagName}`, () => {
  TestTools.testProperties(TagName, CONFIG);
});

describe('HTML render into snapshot', () => {

  it('Blockquote should render as HTML', () => {
    const tree = renderer.create(
      <UU5.Bricks.Blockquote>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit.
      </UU5.Bricks.Blockquote>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

});




