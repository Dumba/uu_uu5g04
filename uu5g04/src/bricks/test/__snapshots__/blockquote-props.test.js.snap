// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HTML render into snapshot Blockquote should render as HTML 1`] = `
<blockquote
  className="uu5-bricks-blockquote"
  style={Object {}}
  title={null}
>
  Lorem ipsum dolor sit amet, consectetur adipiscing elit.
</blockquote>
`;

exports[`UU5.Bricks.Blockquote alignment 1`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote alignment 2`] = `
<blockquote
  className="uu5-bricks-blockquote blockquote-reverse"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote alignment 3`] = `
<blockquote
  className="uu5-bricks-blockquote blockquote-reverse"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote alignment 4`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote background 1`] = `
<blockquote
  className="uu5-bricks-blockquote uu5-bricks-blockquote-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote background 2`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote background 3`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote background 4`] = `
<blockquote
  className="uu5-bricks-blockquote uu5-bricks-blockquote-bg"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote checkGrammar 1`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote checkGrammar 2`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote checkGrammar 3`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote checkGrammar 4`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote checkHighlight 1`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote checkHighlight 2`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote checkHighlight 3`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote checkHighlight 4`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote checkSpaces 1`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote checkSpaces 2`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote checkSpaces 3`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote checkSpaces 4`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote className 1`] = `
<blockquote
  className="uu5-bricks-blockquote my-classname"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 1`] = `
<blockquote
  className="color-schema-default uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 2`] = `
<blockquote
  className="color-schema-red uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 3`] = `
<blockquote
  className="color-schema-red uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 4`] = `
<blockquote
  className="color-schema-red-rich uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 5`] = `
<blockquote
  className="color-schema-red-rich uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 6`] = `
<blockquote
  className="color-schema-green uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 7`] = `
<blockquote
  className="color-schema-green uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 8`] = `
<blockquote
  className="color-schema-green-rich uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 9`] = `
<blockquote
  className="color-schema-green-rich uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 10`] = `
<blockquote
  className="color-schema-yellow uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 11`] = `
<blockquote
  className="color-schema-yellow uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 12`] = `
<blockquote
  className="color-schema-yellow-rich uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 13`] = `
<blockquote
  className="color-schema-yellow-rich uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 14`] = `
<blockquote
  className="color-schema-purple uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 15`] = `
<blockquote
  className="color-schema-purple uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 16`] = `
<blockquote
  className="color-schema-purple-rich uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 17`] = `
<blockquote
  className="color-schema-purple-rich uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 18`] = `
<blockquote
  className="color-schema-brown uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 19`] = `
<blockquote
  className="color-schema-brown uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 20`] = `
<blockquote
  className="color-schema-brown-rich uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 21`] = `
<blockquote
  className="color-schema-brown-rich uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 22`] = `
<blockquote
  className="color-schema-orange uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 23`] = `
<blockquote
  className="color-schema-orange uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 24`] = `
<blockquote
  className="color-schema-orange-rich uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 25`] = `
<blockquote
  className="color-schema-orange-rich uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 26`] = `
<blockquote
  className="color-schema-cyan uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 27`] = `
<blockquote
  className="color-schema-cyan uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 28`] = `
<blockquote
  className="color-schema-cyan-rich uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 29`] = `
<blockquote
  className="color-schema-cyan-rich uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 30`] = `
<blockquote
  className="color-schema-grey uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 31`] = `
<blockquote
  className="color-schema-grey uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 32`] = `
<blockquote
  className="color-schema-grey-rich uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 33`] = `
<blockquote
  className="color-schema-grey-rich uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 34`] = `
<blockquote
  className="color-schema-pink uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 35`] = `
<blockquote
  className="color-schema-pink uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 36`] = `
<blockquote
  className="color-schema-pink-rich uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 37`] = `
<blockquote
  className="color-schema-pink-rich uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote colorSchema 38`] = `
<blockquote
  className="color-schema-pink uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote content 1`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
>
  12345
</blockquote>
`;

exports[`UU5.Bricks.Blockquote content 2`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
>
  <UU5.Common.TextCorrectorContextConsumer
    key="0"
    text="This is Content from ContentMixin"
  />
</blockquote>
`;

exports[`UU5.Bricks.Blockquote content 3`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
>
  <UU5.Common.TextCorrectorContextConsumer
    key="0"
    text="This is Content from ContentMixin"
  />
</blockquote>
`;

exports[`UU5.Bricks.Blockquote content 4`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
>
  12345
</blockquote>
`;

exports[`UU5.Bricks.Blockquote controlled 1`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote controlled 2`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote controlled 3`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote controlled 4`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote default props 1`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote disabled 1`] = `
<blockquote
  className="uu5-bricks-blockquote uu5-common-disabled"
  id="uuID"
  style={Object {}}
  title={null}
>
  <span
    className="uu5-common-disabled-cover"
    key="disabledCover"
  />
</blockquote>
`;

exports[`UU5.Bricks.Blockquote disabled 2`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote disabled 3`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote disabled 4`] = `
<blockquote
  className="uu5-bricks-blockquote uu5-common-disabled"
  id="uuID"
  style={Object {}}
  title={null}
>
  <span
    className="uu5-common-disabled-cover"
    key="disabledCover"
  />
</blockquote>
`;

exports[`UU5.Bricks.Blockquote dynamic 1`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote dynamic 2`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote dynamic 3`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote dynamic 4`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote footer 1`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote footerAlignment 1`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote footerAlignment 2`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote footerAlignment 3`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote footerAlignment 4`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote hidden 1`] = `
<blockquote
  className="uu5-bricks-blockquote uu5-common-hidden"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote hidden 2`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote hidden 3`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote hidden 4`] = `
<blockquote
  className="uu5-bricks-blockquote uu5-common-hidden"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote id 1`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="myId"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote ignoreHTML 1`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote ignoreHTML 2`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote ignoreHTML 3`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote ignoreHTML 4`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote mainAttrs 1`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote name 1`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  name="myName"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote nestingLevel 1`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote nestingLevel 2`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote nestingLevel 3`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote nestingLevel 4`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote nestingLevel 5`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote nestingLevel 6`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote nestingLevel 7`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote nestingLevel 8`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote nestingLevel 9`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote nestingLevel 10`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote nestingLevel 11`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote nestingLevel 12`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote nestingLevel 13`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote nestingLevel 14`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote nestingLevel 15`] = `""`;

exports[`UU5.Bricks.Blockquote nestingLevel 16`] = `""`;

exports[`UU5.Bricks.Blockquote nestingLevel 17`] = `""`;

exports[`UU5.Bricks.Blockquote nestingLevel 18`] = `""`;

exports[`UU5.Bricks.Blockquote nestingLevel 19`] = `""`;

exports[`UU5.Bricks.Blockquote nestingLevel 20`] = `""`;

exports[`UU5.Bricks.Blockquote noIndex 1`] = `
<blockquote
  className="uu5-bricks-blockquote uu5-noindex"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote noIndex 2`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote noIndex 3`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote noIndex 4`] = `
<blockquote
  className="uu5-bricks-blockquote uu5-noindex"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote noSpacing 1`] = `
<blockquote
  className="uu5-bricks-blockquote uu5-bricks-blockquote-nospacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote noSpacing 2`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote noSpacing 3`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote noSpacing 4`] = `
<blockquote
  className="uu5-bricks-blockquote uu5-bricks-blockquote-nospacing"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote pureRender 1`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote pureRender 2`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote pureRender 3`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote pureRender 4`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote selected 1`] = `
<blockquote
  className="uu5-bricks-blockquote uu5-common-selected"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote selected 2`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote selected 3`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote selected 4`] = `
<blockquote
  className="uu5-bricks-blockquote uu5-common-selected"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote style 1`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={
    Object {
      "borderRadius": "2px",
      "padding": "8px",
    }
  }
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote style 2`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={
    Object {
      "backgroundColor": "red",
      "color": "blue",
    }
  }
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote style 3`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={
    Object {
      "backgroundColor": "red",
      "color": "blue",
    }
  }
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote style 4`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={
    Object {
      "borderRadius": "2px",
      "padding": "8px",
    }
  }
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote textCorrector 1`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote textCorrector 2`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote textCorrector 3`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote textCorrector 4`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title={null}
/>
`;

exports[`UU5.Bricks.Blockquote tooltip 1`] = `
<blockquote
  className="uu5-bricks-blockquote"
  id="uuID"
  style={Object {}}
  title="My Tooltip"
/>
`;
